src/components/map/map-header.tsx

```diff
@@ -2,61 +2,55 @@ import style from './map.module.css';
 
 import { ButtonLink } from '@atb/components/button';
 import { ComponentText, useTranslation } from '@atb/translations';
-import VenueIcon, { FeatureCategory } from '@atb/components/venue-icon';
 import { and } from '@atb/utils/css';
 import { MonoIcon } from '@atb/components/icon';
+import { TransportMode } from '@atb/components/transport-mode/types';
+import { transportModeToTranslatedString } from '@atb/components/transport-mode';
+import { getTransportModeIcon } from '@atb/components/transport-mode/transport-icon';
 
 export type MapHeaderProps = {
   id: string;
   name: string; // StopPlace name or address
   layer: 'address' | 'venue';
-  street?: string;
-  category?: FeatureCategory[];
+  transportModes?: TransportMode[];
 };
 
-export function MapHeader({
-  id,
-  name,
-  layer,
-  street,
-  category,
-}: MapHeaderProps) {
+export function MapHeader({ id, name, layer, transportModes }: MapHeaderProps) {
   const { t } = useTranslation();
   return (
     <div className={style.header}>
       <div className={style.header__leftContainer}>
         <div className={style.header__icons}>
-          {layer === 'address' || !category ? (
+          {layer === 'address' || !transportModes ? (
             <div>
               <MonoIcon size="large" icon="map/Pin" overrideMode="dark" />
             </div>
           ) : (
-            category.map((type) => (
-              <div key={[type, 'icon'].join('-')}>
-                <VenueIcon category={[type]} size="large" overrideMode="dark" />
+            transportModes.map((mode) => (
+              <div key={[mode, 'icon'].join('-')}>
```

Comment:
Is this shorter and more intuitive than this?
```suggestion
              <div key={`${mode}-icon`}>
```

Reply:
(he asks almost passive-aggressively and rhetorically)

Reply:
😄 I agree though